docs/source/scripts.rst

```diff
@@ -0,0 +1,56 @@
+..  scripts:
+
+Utility scripts
+===============
+
+Visualize data augmentation transformations
```

Comment:
not sure how they appear on the left TOC, but it is fairly small, so i would keep titles short, e.g. "Visualize transformations" or "Visualize augmentation transfos"
but this is a very minor detail...

Reply:
update: after looking at the generated built doc, it looks fine the way it is 👍